src/ophyd_async/plan_stubs/_fly.py

```diff
@@ -74,24 +72,26 @@ def prepare_static_seq_table_flyer_and_detectors_with_same_trigger(
     trigger_time = number_of_frames * (exposure + deadtime)
     pre_delay = max(period - 2 * shutter_time - trigger_time, 0)
 
-    table: SeqTable = seq_table_from_rows(
+    table = (
         # Wait for pre-delay then open shutter
-        SeqTableRow(
+        SeqTable.row(
             time1=in_micros(pre_delay),
```

Comment:
... as we have an implicitly defined `repeats` field that will have changes defaults from `1` to `0`. Are we lacking a test for this plan stub?

Reply:
We do have tests for this but at no point in them do we check on `repeats`